README.md

```diff
@@ -1,202 +1,66 @@
-* [Goya](http://jackschaedler.github.io/goya/), pixel editor with
-  undo/redo and visual history
-* [AppShare](https://github.com/zubairq/AppShare), a Clojure web framework
-* [wordsmith](http://wordsmith.variadic.me), a markdown editor
-* [omchaya](http://github.com/sgrove/omchaya)
-* [BVCA Private Equity Map](http://bvca.clustermap.trampolinesystems.com/)
-* [session](http://github.com/kovasb/session)
-* [pOModoro](http://pomodoro.trevorlandau.net)
-* [Dakait](http://github.com/verma/dakait), a web-based tool to manage
-  downloads
-* [Mega Super Mario World](http://github.com/city41/mario-review), a detailed review of the classic video game and a SNES video editor
-* [Time for Coffee!](http://www.timeforcoffee.ch), a handy website to display the next departures at public transport stops in Switzerland
-* [Omingard](https://omingard.5apps.com), a Solitaire-like card game. Making-of: [My Way into Clojure](http://www.railslove.com/stories/my-way-into-clojure-building-a-card-game-with-om-part-1).
-* [Horizon Alpha](https://github.com/BertrandDechoux/horizon-alpha), a quick Hack and slash game using the Noob universe
-* [Solari Architects](http://solariarchitects.com/), portfolio for architecture firm.  
-
-## Using it
-
-The current version depends on React 0.13.3.
-
-Make sure you have [Leiningen](http://leiningen.org/) installed.
-
-Your `project.clj` should include something like the following:
-
-```clojure
-(defproject foo "0.1.0"
-  ...
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.clojure/clojurescript "0.0-2760"]
-                 [org.omcljs/om "0.9.0"]]
-  ...)
+```
+<dependency>
+  <groupId>org.omcljs</groupId>
+  <artifactId>om</artifactId>
+  <version>1.0.0-beta1</version>
+</dependency>
 ```
 
-### React with Add-Ons
-
-If you would rather use React with Add-Ons you can configure this
-with Maven's exclusions feature:
+## Example
 
 ```clojure
-(defproject foo "0.1.0"
-  ...
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.clojure/clojurescript "0.0-2760"]
-                 [org.omcljs/om "0.9.0" :exclusions [cljsjs/react]]
-                 [cljsjs/react-with-addons "0.13.3-0"]]
-  ...)
-```
+(ns example
+  (:require [om.dom :as dom]
+            [om.next :as om]))
 
-### Build configuration
+(defui Widget
+  om/IRender
+  (render [this]
+    (dom/h1 nil (:text data)))
 
-For local development your
-[lein-cljsbuild](http://github.com/emezeske/lein-cljsbuild) settings
-should look something like this:
+(def widget (om/factory Widget)
 
-```clojure
-:cljsbuild {
-  :builds [{:id "dev"
-            :source-paths ["src"]
-            :compiler {
-              :main main.core
-              :output-to "main.js"
-              :output-dir "out"
-              :optimizations :none
-              :source-map true}}]}
+(om/root widget {:text "Hello world!"}
+  {:target (. js/document (getElementById "my-app"))})
 ```
 
-Your markup should look something like the following:
+## Tutorials
```

Comment:
I think we can also link to the entire Om Next section in the Wiki?
Link: https://github.com/omcljs/om/wiki#om-next